lib/app/presenters/docs.rb

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+module ExercismWeb
+  module Presenters
+    class Doc < OpenStruct
+    end
+
+    class Docs
+      def initialize(track_id)
+        @track_id = track_id
+      end
+
+      def self.fetch_docs
+        status, body = Xapi.get("docs/#{@track_id}")
```

Comment:
I think I'd like to do the actual fetching within the route, and then pass the body of the response to the presenter, so that it can hide some of the complexity of munging the data.